tap_gsheets/__init__.py

```diff
@@ -64,9 +70,13 @@ def process_worksheet(gsheets_loader, sheet_name, worksheet, start_from_row, con
     else:
         stream_name = tableize(parameterize(name_with_worksheet))
 
+    records = gsheets_loader.get_records_as_json(sheet_name, worksheet, start_from_row)
+
+    if gsheet_loader.date_format:
```

Comment:
If I followed the logic correctly, this field will _always_ exist, right? If that is the case, then this `if` is unnecessary. 🙂

Reply:
if we need to transform date column we will set date template to this field otherwise it will empty all the time